### PR TITLE
Recruitment regions and trusts

### LINKED ISSUE #265: Add graphic to show regions on Fellows landing page
At the moment the fellows landing page isnt super clear that it is a national programme. i think it would help to have a inforgraphic showing the regions that are included just as a rough visual to anchor people. Here's an example from a paper I wrote showing the sites we're recruiting from:
image
What do you think @Beatrix-Fletcher @alexandertdeng ?

### PR DESCRIPTION
**Pull request by alexandertdeng**
Text version, graphic for landing page in progress

### CODE REVIEW

File: _pages/fellowship_cohort_applications.md [hunk 1]
```diff
@@ -6,6 +6,10 @@ title: Fellowship in Clinical AI - Cohort 3 Application (2024-25)
 Applications for Cohort 3 of the Fellowship in Clinical AI are now open.
 Successful applicants will be fellows in clinical AI from August 2024 - August 2025.
 
+<h3 style>Recruitment Regions</h3>
+![Recruitment region map](/assets/img/people/FCAI_Cohort3_map.png)
```
Comment (AnilMistry) [inline _pages/fellowship_cohort_applications.md]: i think this image looks too big on the page, i'd suggest replacing with :
`<div style="text-align: center"></div><br><br>`

File: _pages/fellowship_general.md [hunk 2]
```diff
@@ -3,6 +3,38 @@ layout: people
 permalink: /fellowship_general.html
 title: Fellowship General Information
 ---
+<h3 style>NHS Recruitment Regions and AI Project Host Trusts</h3>
+
+* England
+  * East of England
+    * Cambridge University Hospitals
+  * London
+    * Guy's and St Thomas's
```
Comment (AnilMistry) [inline _pages/fellowship_general.md]: Guy's and St Thomas'

**Review by AnilMistry — CHANGES_REQUESTED** [review]
minor changes

File: _pages/fellowship_general.md (hunk 2 above)
Comment (alexandertdeng) [reply _pages/fellowship_general.md]: Both above changes made- I have moved the regional information down the page in both cases as it's a secondary consideration compared to the other information on the page

**Review by AnilMistry — APPROVED** [review]

**Pull request merged**